src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -1,33 +1,31 @@
-﻿<?xml version="1.0"?>
+<?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Orleans.CounterControl</id>
```

Comment:
@jason-bragg  I'm against that... Why should we keep it? The command line is basically the same. There is no need to deploy that .exe anymore and if you need PerfCounter the dll will be there deployed with server code. The user can still use the old CounterControl nuget to register if they want. We are just not going to push anymore new nugets so why keep the project? I though by your [comment ](https://github.com/dotnet/orleans/pull/2122#discussion_r78265899) here that you agree with me and if I add that to this PR we could remove the old one... Its totally pointless to have it if you leave the old nuget alive...